dist/src/main/config/zeebe.cfg.toml

```diff
@@ -30,6 +30,41 @@
 
 # ----------------------------------------------------
 
+
+[gateway]
+# Enable the embedded gateway to start on broker startup
+#
+# This setting can also be overridden using the environment variable ZEEBE_EMBED_GATEWAY.
+# enable = true
+
+[gateway.network]
+# Sets the host the embedded gateway binds to
+#
+# This setting can also be overridden using the environment variable ZEEBE_GATEWAY_HOST.
+# host = "0.0.0.0"
+#
+# Sets the port the embedded gateway binds to
+#
+# This setting can also be overridden using the environment variable ZEEBE_GATEWAY_PORT.
+# port = 26500
+
+[gateway.cluster]
+# Sets the broker the gateway should initial contact
+#
+# This setting can also be overridden using the environment variable ZEEBE_GATEWAY_CONTACT_POINT.
+# contactPoint = "127.0.0.1:26501"
+
+# Sets size of the transport buffer to send and received messages between gateway and broker cluster
+#
```

Comment:
How large the transport buffer should be? Maybe we can explain in the config our properties a bit more in detail what this means. What do you think?